README.md

```diff
@@ -138,30 +151,29 @@ Congratulations! You have successfully started the flow. Carry on reading the ne
 - Handle callbacks
 - Remove the SDK from the page
 - Customise the SDK
-- Upload files to the Onfido API
 - Create checks
 
 ## Handling callbacks
 
-A number of callback functions are fired at various points of the flow. The most important function is `onComplete`. Inside this callback, you would typically send the captured images to your backend server, where you would then [upload files](#uploading-files-to-onfido-api) and [create checks](#creating-checks) using the [Onfido API](https://documentation.onfido.com/).
+A number of callback functions are fired at various points of the flow. The most important function is `onComplete`. Inside this callback you would typically trigger your backend to create a check using the [Onfido API](https://documentation.onfido.com/).
 
 - **`onReady {Function} optional`**
 
   Callback function that fires once the library has successfully authenticated using the JWT. In this function we recommend removing the `disabled` attribute on the modal trigger button.
 
 - **`onDocumentCapture {Function} optional`**
 
-  Callback that fires when the document has been successfully captured and confirmed by the user. It returns an object that contains the document capture.
+  Callback that fires when the document has been successfully captured, confirmed by the user and uploaded to the Onfido API. It returns an object that contains the document capture.
 
 - **`onFaceCapture {Function} optional`**
 
-  Callback that fires when the face has been successfully captured and confirmed by the user. It returns an object that contains the face capture.
+  Callback that fires when the face has been successfully captured, confirmed by the user and uploaded to the Onfido API. It returns an object that contains the face capture.
 
 - **`onComplete {Function} optional`**
 
-  Callback that fires when both the document and face have successfully been captured. It returns an object that contains both captures. This event data should sent to your backend where the full [API requests](#uploading-files-to-onfido-api) will be made.
+  Callback that fires when both the document and face have successfully been captured and uploaded. It returns an object that contains both captures. This event data should sent to your backend where the full [API requests](https://documentation.onfido.com/#create-check) will be made.
```

Comment:
This paragraph needs updating. They do not need to send any data to the backend - they just need to trigger the backend to create a check.

Also `both captures` should read `all captures` because there could be multiple documents